### PR TITLE
[docs content] Image README formatting

### PR DESCRIPTION
**Pull request by sarah11918**
## Changes

Changes some code formatting in 2 places to fix integration with Docs. Failing Docs checks, and I think it's do do with `'` used instead of `` ` ``

## Testing

Not tested! Not even sure I got the second one right. But, these would seem to be the potential problem areas causing errors. For @tony-sull to review!  😄 

## Docs

Will auto-update Docs integration content

### CODE REVIEW

File: packages/integrations/image/README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ For images in your project's repository, use the `src` relative to the `public`
 
 <p>
 
-**Type:** 'avif' | 'jpeg' | 'png' | 'webp'<br>
+**Type:** `avif` | `jpeg` | `png` | `webp`<br>
```
Comment (tony-sull) [inline packages/integrations/image/README.md:105]: Oh interesting, looking at the auto-gen for the Configuration Reference it looks like maybe this should be:

```
**Type:** `'avif' | 'jpeg' | 'png' | 'webp'`<br>
```

Not sure if that would render out exactly the same on the integration pages though

Comment (sarah11918) [reply packages/integrations/image/README.md:105]: Oh, that could be, too! I'm honestly not sure myself, but looking at the README just on GitHub, I can see it's the two sections using avif etc. where the rendering looks different.

**Review by matthewp — APPROVED** [review]

File: packages/integrations/image/README.md (hunk 1 above)
Comment (delucis) [reply packages/integrations/image/README.md:105]: Yup, we should keep the quotes around each item (indicates a TypeScript string literal) here but wrap the whole union in backticks:

```suggestion
**Type:** `'avif' | 'jpeg' | 'png' | 'webp'`<br>
```

File: packages/integrations/image/README.md [hunk 2]
```diff
@@ -229,7 +229,7 @@ A `number` can also be provided, useful when the aspect ratio is calculated at b
 
 <p>
 
-**Type:** Array<'avif' | 'jpeg' | 'png' | 'webp'><br>
+**Type:** `Array<`avif` | `jpeg` | `png` | `webp`>`<br>
```
Comment (delucis) [inline packages/integrations/image/README.md:232]: Same here:

```suggestion
**Type:** `Array<'avif' | 'jpeg' | 'png' | 'webp'>`<br>
```

**Review by delucis — COMMENTED** [review]
Made suggestions for how best to fix this. I’d say get this merged as soon as these are made to avoid blocking updates to the integration pages in docs any longer!

**Comment by sarah11918** [comment]
Thanks, @delucis , for checking the docs build!

**Pull request merged**